src/main/kotlin/no/nav/k9/pleiepengersyktbarn/soknad/PleiepengerSyktBarnRoutes.kt

```diff
@@ -181,4 +196,68 @@ internal class PleiepengerSyktBarnRoutes(
     }
     private suspend fun ServerRequest.innsending() = body(BodyExtractors.toMono(Innsending::class.java)).awaitFirst()
     private fun ServerRequest.mappeLocation(mappeId: MappeId) = uriBuilder().pathSegment("mappe", mappeId).build()
+
+    private fun søknadConverter(pleiepengerSyktBarnSoknad: PleiepengerSyktBarnSoknad?, ident: NorskIdent): PleiepengerBarnSøknad {
```

Comment:
Denne funksjonen kan kanskje legges inn en egen fil, et `object` så det ikke er for mye inne Routes.
Og hvorfor er søknaden optional?

Reply:
Godt poeng! Kan ikke se noen grunn til at den skal være optional; det er sikkert bare noe som henger igjen fra feilsøking eller noe. Fikser det!